openslides/agenda/signals.py

```diff
@@ -15,65 +15,50 @@ def validate_start_time(value):
     try:
         datetime.strptime(value, '%d.%m.%Y %H:%M')
     except ValueError:
-        raise ValidationError(_('Invalid input.'))
+        raise DjangoValidationError(_('Invalid input.'))
```

Comment:
django restframework recomments this: https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/exceptions.py#L55